cdc/api/owner.go

```diff
@@ -79,13 +75,10 @@ type ownerAPI struct {
 	capture *capture.Capture
 }
 
-// RegisterOwnerAPIRoutes registers routes for owner APIs.
-func RegisterOwnerAPIRoutes(router *gin.Engine, capture *capture.Capture) {
+// registerOwnerAPIRoutes registers routes for owner APIs.
+func registerOwnerAPIRoutes(router *gin.Engine, capture *capture.Capture) {
 	ownerAPI := ownerAPI{capture: capture}
-	router.POST("/capture/owner/resign", gin.WrapF(ownerAPI.handleResignOwner))
 	router.POST("/capture/owner/admin", gin.WrapF(ownerAPI.handleChangefeedAdmin))
-	router.POST("/capture/owner/rebalance_trigger", gin.WrapF(ownerAPI.handleRebalanceTrigger))
-	router.POST("/capture/owner/move_table", gin.WrapF(ownerAPI.handleMoveTable))
```

Comment:
It's kind of breaking change if we remove them directly. Could you make them as alias to the new open api?